docs/orchestrating-elastic-stack-applications/apm-server.asciidoc

```diff
@@ -106,6 +106,21 @@ spec:
 EOF
 ----
 
+Since stack version 8.0.0, along with specifying the `kibanaRef`, you also need to configure Kibana to include the APM integration. Add the following section to the Kibana specification:
```

Comment:
This seems misplaced in the APM Agent central configuration section which is way to configure your APM agents from Kibana. I think this has to go into the section where we first explain how to deploy APM Server `Use an Elasticsearch cluster managed by ECK` If I understand correctly APM Server now requires a Kibana since 8.0.0 with the integration so it should be mentioned there that both are needed.

Reply:
We discussed and I removed the doc changes, as we'll handle this when we tackle #7541